src/Core/System/StateMachine/Aggregation/StateMachineTransition/StateMachineTransitionActions.php

```diff
@@ -11,8 +11,8 @@ final class StateMachineTransitionActions
     public const ACTION_COMPLETE = 'complete';
     public const ACTION_DO_PAY = 'do_pay';
     public const ACTION_FAIL = 'fail';
-    public const ACTION_PAID = 'paid';
-    public const ACTION_PAID_PARTIALLY = 'paid_partially';
+    public const ACTION_PAID = 'pay';
```

Comment:
why do you think, those are invalid? where and how do you get an error? 

the values of the constants where like this in the past, and were changed to those

Reply:
In the state_machine_transition table are inconsistent entries for the state machine "order_transaction.state". So it has entries with action_names:
* pay
* paid

The error comes, to set an order transition as paid. The order transition was in state paid_partially.
In the Screenshot of the transition table, you can see, that the "paid" action has no available transitions to set an order transition from "paid_partially" to "paid"
![Bildschirmfoto 2024-02-19 um 11 35 04](https://github.com/shopware/shopware/assets/6031484/ecb4c802-61ec-4876-8450-aa957e795dd5)
. Only the action "pay" has a valid transition entry.

Reply:
As @mitelg wrote, unfortunately, the naming of the actions is a mess right now. We have it on our agenda to fix this for the next major, 6.7, but it would be great, if you could get a head start on this and add a migration for this.

I think, the transition names should be as following (and the constants therefore as well):
- `pay` (instead of `paid`)
- `process` (instead of `do_pay`)
- `pay_partially` (instead of `paid_partially`)

Please remember to not delete constants as this would break this. Deprecate constants with `@deprecated - tag:v6.7.0 - use XYZ instead` and assign their values to the new constants.